app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,42 @@
+  <nav class="navbar navbar-expand-lg bg-body-tertiary shadow">
+  <div class="container py-2">
+    <%= link_to "Recipe", unauthenticated_root_path, class: "navbar-brand me-5 border text-primary p-2 bg-light bg-gradient rounded-circle" %>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item">
+          <%= link_to "Recipes", recipes_path, class: "nav-link" %>
+        </li>
+
+        <li class="nav-item">
+          <%= link_to "Foods", foods_path, class: "nav-link" %>
+        </li>
+
+        <li class="nav-item">
+          <%= link_to "Public recipes", public_recipes_path, class: "nav-link" %>
+        </li>
+      </ul>
+
```

Comment:
:red_circle: Please make sure the General shopping list is also accessible from the navbar menu.